tfjs-core/src/ops/browser.ts

```diff
@@ -155,7 +155,11 @@ function fromPixels_(
               'Reason: OffscreenCanvas Context2D rendering is not supported.');
         }
       } else {
-        fromPixels2DContext = document.createElement('canvas').getContext('2d');
+        const willReadFrequently =
+            env().getBool('CANVAS2D_WILL_READ_FREQUENTLY');
+        const options = willReadFrequently ? {willReadFrequently} : {};
+        fromPixels2DContext = document.createElement('canvas').getContext(
```

Comment:
nit: It seems that you can directly write it like below and remove options. Tested [here](https://github.com/tensorflow/tfjs/pull/6486).
```
        fromPixels2DContext =
            document.createElement('canvas').getContext(
                '2d', {willReadFrequently}) as CanvasRenderingContext2D;